SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,31 +1,22 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
+# SpatialOS GDK for Unreal toolbar
```

Comment:
Haven't looked at this but there's some other edits going on elsewhere. I don't think this has them?!